src/lib/executors/Node.ts

```diff
@@ -886,7 +903,8 @@ export default class Node extends Executor<NodeEvents, Config, NodePlugins> {
     );
     this._unhookRequire = hookRequire(
       filename => this.shouldInstrumentFile(filename),
-      (code, { filename }) => this.instrumentCode(code, filename)
+      (code, { filename }) => this.instrumentCode(code, filename),
+      { extensions: ['.js', '.jsx', '.ts', 'tsx'] }
```

Comment:
What about `mjs`?

Reply:
The default was only `.js` (or at least that's what I gathered looking at the code, didn't see anywhere it was documented).  Even after registering tsnode `.mjs` doesn't exist in `require.extensions` so I'm not sure we should or even could add that.

Reply:
Thanks for looking into that. We can always get back to it later if it's an issue.